scss/mixins.scss

```diff
@@ -290,122 +131,33 @@ $_o-brand-default-not-set-warning: false !default; // Has the default not set wa
     }
     // Validate brand variables and supported variants.
     $variables: map-get($config, 'variables');
-    $supports-variants: if(map-get($config, 'settings'), map-get($config, 'settings'), map-get($config, 'supports-variants'));
+    $supports-variants: map-get($config, 'supports-variants');
     @if $variables and (type-of($variables) != 'map' and type-of($variables) != 'list' ) {
         @error '#{$errorMessage}. Config key "variables" should be a map but is of type #{type-of($variables)}.';
```

Comment:
The check here makes me think a `list` is fine to use but the error message specifically states that a `map` should be used, which is it?

Reply:
When the configuration is invalid, we should include a link to the brand configuration documentation in the error message.

Reply:
An empty map is a list `()`. Added a comment to clarify.